utils/install_dag.sh

```diff
@@ -33,6 +39,12 @@ AIRFLOW_DAG_PATH=${AIRFLOW_HOME}/dags/${IMAGE}
 CONTAINER_DAG_PATH=/dags
 POST_INSTALL=${AIRFLOW_DAG_PATH}/post_install.sh
 
+
+while ! wait_for_docker_daemon; do
+  echo "Waiting until Docker Daemon is ready."
+  sleep 5
```

Comment:
This is currently retrying the connection to the docker daemon until it succeeds. Please add some maximum amount of times to retry, so that if there's a problem with the docker daemon this eventually fails and the pod is restarted.